staging/src/k8s.io/apiserver/pkg/storage/tests/cacher_test.go

```diff
@@ -100,6 +106,10 @@ func newEtcdTestStorage(t *testing.T, prefix string) (*etcd3testing.EtcdTestServ
 }
 
 func newTestCacher(s storage.Interface, cap int) (*cacherstorage.Cacher, storage.Versioner, error) {
+	return buildCacherWithClock(s, cap, clock.RealClock{})
+}
+
+func buildCacherWithClock(s storage.Interface, cap int, clock clock.Clock) (*cacherstorage.Cacher, storage.Versioner, error) {
```

Comment:
nit: let's name it newTestCacherWithClock

Reply:
done.